### DISCUSSION TITLE
[SF][A11Y]: AnalyticalTable content can't be read by JAWs

### DISCUSSION THREAD

**Discussion by JosephineSu** [UI5/webcomponents-react#5511] [Q&A]
Hi Colleagues,
There is an announcement issue reported by custom: AnalyticalTable content can't be read out by JAWS(not the latest version).
See below screenshot, suppose the Name column is clickable, when "Allen Best" is selected, only "Name" read out, which is not expected. We notice role="grid" has been used, and this will disrupt screen reading software. Is it possible to change this role? 
Or any other suggestions?
<img width="1039" alt="Screenshot 2024-02-20 at 17 29 18" src="https://github.com/SAP/ui5-webcomponents-react/assets/38159212/389e26b9-03bc-4c31-894c-458927a0f022">

<img width="1175" alt="Screenshot 2024-02-20 at 17 30 21" src="https://github.com/SAP/ui5-webcomponents-react/assets/38159212/56a867ec-ee09-48f3-9347-6f5e94328e1c">

Thanks,
Xiaohui Su

**Comment by Lukas742** [comment]
Hi @JosephineSu 

grid tables are widely support and therefore `role="grid"` should not be a problem. At first glance I think that if the pop-in feature is used, aria announcements are not set correctly, but we have to investigate that further. Since this is a bug report and not a question I will close it here. Please create a [bug report](https://github.com/SAP/ui5-webcomponents-react/issues/new?assignees=&labels=bug&projects=&template=bug-report.yml&title=%5BComponent+Name%5D%3A+%5BBug+Description%5D) using our issue template.
